lib/launcher.js

```diff
@@ -329,6 +329,22 @@ class Launcher {
             }
         }
 
+        if (this.config.localAuth?.enabled) {
+            settings.localAuth = {
+                enabled: true,
+                user: this.config.localAuth.user,
+                pass: this.config.localAuth.pass
+            }
+        } else if (this.settings?.security?.localAuth.enabled === true) {
```

Comment:
```suggestion
        } else if (this.settings?.security?.localAuth?.enabled === true) {
```

Hit an error with an existing device when running for the first time.